src/cli/pubkey.cpp

```diff
@@ -215,6 +217,15 @@ class PK_Sign final : public Command
          this->read_file(get_arg("file"), onData);
 
          output() << Botan::base64_encode(signer.signature(rng())) << "\n";
```

Comment:
At this point a valid signature with the current leaf index is generated and immediately written to stdout...